scripts/install.sh

```diff
@@ -81,30 +76,68 @@ After=network-online.target
 
 [Service]
 ExecStart=$BINDIR/ollama serve
-User=ollama
-Group=ollama
+User=$(whoami)
+Group=$(id -gn)
 Restart=always
 RestartSec=3
-Environment="HOME=/usr/share/ollama"
+Environment="HOME=$(eval echo ~$(whoami))"
 Environment="PATH=$PATH"
 
 [Install]
 WantedBy=default.target
 EOF
+
     SYSTEMCTL_RUNNING="$(systemctl is-system-running || true)"
     case $SYSTEMCTL_RUNNING in
         running|degraded)
             status "Enabling and starting ollama service..."
             $SUDO systemctl daemon-reload
-            $SUDO systemctl enable ollama
+            $SUDO systemctl enable ollama > /dev/null 2>&1
```

Comment:
Why is this being blackholed?

Reply:
I was seeing log that seemed not useful from this, but I've un-blackholed it now